### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L te,counterfit,afile` and `typos --hidden --format brief`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ Projects using cmd2
 | [Poseidon](https://github.com/CyberReboot/poseidon)             | Leverages software-defined networks (SDNs) to acquire and then feed network traffic to a number of machine learning techniques. |   |
 | [Unipacker](https://github.com/unipacker/unipacker)             | Automatic and platform-independent unpacker for Windows binaries based on emulation                                             |   |
 | [tomcatmanager](https://github.com/tomcatmanager/tomcatmanager) | A command line tool and python library for managing a tomcat server                                                             |   |
-| [Expliot](https://gitlab.com/expliot_framework/expliot)         | Internet of Things (IoT) exploitation framework                                                                                 |   |
+| [Exploit](https://gitlab.com/expliot_framework/expliot)         | Internet of Things (IoT) exploitation framework                                                                                 |   |
```
Comment (kmvanbrunt) [inline README.md:156]: "Expliot" is the name of the software. It's not a typo.

File: README.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ Projects using cmd2
 | [Poseidon](https://github.com/CyberReboot/poseidon)             | Leverages software-defined networks (SDNs) to acquire and then feed network traffic to a number of machine learning techniques. |   |
 | [Unipacker](https://github.com/unipacker/unipacker)             | Automatic and platform-independent unpacker for Windows binaries based on emulation                                             |   |
 | [tomcatmanager](https://github.com/tomcatmanager/tomcatmanager) | A command line tool and python library for managing a tomcat server                                                             |   |
-| [Expliot](https://gitlab.com/expliot_framework/expliot)         | Internet of Things (IoT) exploitation framework                                                                                 |   |
```
Comment (tleonhardt) [inline README.md:156]: This is the name of the package and not a typo

File: readme_files/shout_out.csv [hunk 3]
```diff
@@ -5,7 +5,7 @@ Application Name, Description
 [Poseidon](https://github.com/CyberReboot/poseidon),Leverages software-defined networks (SDNs) to acquire and then feed network traffic to a number of machine learning techniques.
 [Unipacker](https://github.com/unipacker/unipacker),Automatic and platform-independent unpacker for Windows binaries based on emulation
 [tomcatmanager](https://github.com/tomcatmanager/tomcatmanager),A command line tool and python library for managing a tomcat server
-[Expliot](https://gitlab.com/expliot_framework/expliot),Internet of Things (IoT) exploitation framework
+[Exploit](https://gitlab.com/expliot_framework/expliot),Internet of Things (IoT) exploitation framework
```
Comment (kmvanbrunt) [inline readme_files/shout_out.csv:8]: "Expliot" is the name of the software. It's not a typo.

Comment (tleonhardt) [inline readme_files/shout_out.csv:8]: Same name of package not a typo

File: readme_files/shoutout.txt [hunk 4]
```diff
@@ -6,10 +6,10 @@ Genre:
 
 
 
-Name: Microsoft/ Azure Counterfit
+Name: Microsoft/ Azure Counterfeit
```
Comment (tleonhardt) [inline readme_files/shoutout.txt:9]: Also name of a package

Comment (kmvanbrunt) [inline readme_files/shoutout.txt:9]: "Azure Counterfit" is the name of the software. It's not a typo.

File: readme_files/shoutout.txt [hunk 5]
```diff
@@ -6,10 +6,10 @@ Genre:
...
 Source Code: https://github.com/Azure/counterfit
 Status: Active
-Description: Counterfit is a command-line tool and generic automation layer for assessing the security of machine learning systems.
+Description: Counterfeit is a command-line tool and generic automation layer for assessing the security of machine learning systems.
```
Comment (kmvanbrunt) [inline readme_files/shoutout.txt:12]: "Azure Counterfit" is the name of the software. It's not a typo.

**Review by tleonhardt — CHANGES_REQUESTED** [review]

File: tests/test_utils.py [hunk 6]
```diff
@@ -876,16 +876,16 @@ def test_find_editor_not_specified():
 
 
 def test_similarity():
-    suggested_command = cu.suggest_similar("comand", ["command", "UNRELATED", "NOT_SIMILAR"])
+    suggested_command = cu.suggest_similar("command", ["command", "UNRELATED", "NOT_SIMILAR"])
```
Comment (kmvanbrunt) [inline tests/test_utils.py:879]: This unit test requires "command" to be misspelled. It's not a typo.

File: tests/test_utils.py [hunk 7]
```diff
@@ -876,16 +876,16 @@ def test_find_editor_not_specified():
...
     assert suggested_command == "command"
     suggested_command = cu.suggest_similar("command", ["COMMAND", "acommands"])
     assert suggested_command == "COMMAND"
 
 
 def test_similarity_without_good_canididates():
-    suggested_command = cu.suggest_similar("comand", ["UNRELATED", "NOT_SIMILAR"])
+    suggested_command = cu.suggest_similar("command", ["UNRELATED", "NOT_SIMILAR"])
```
Comment (kmvanbrunt) [inline tests/test_utils.py:886]: This unit test requires "command" to be misspelled. It's not a typo.

File: tests/test_utils.py [hunk 8]
```diff
@@ -876,16 +876,16 @@ def test_find_editor_not_specified():
...
     assert suggested_command is None
-    suggested_command = cu.suggest_similar("comand", [])
+    suggested_command = cu.suggest_similar("command", [])
```
Comment (kmvanbrunt) [inline tests/test_utils.py:888]: This unit test requires "command" to be misspelled. It's not a typo.

**Review by kmvanbrunt — CHANGES_REQUESTED** [review]
Thank you for this pull request. I appreciate you fixing our typos.
However, a few of the changes are incorrect. Please make the requested changes.

**Comment by tleonhardt** [comment]
@kianmeng Thanks for this PR. We appreciate the typos you fixed. If you can address comments from the two of us to fix the things which actually are not typos, we would be happy to merge this in.  Thanks again!

**Comment by kmvanbrunt** [comment]
Closing since I'm doing this in another branch.

**Comment by kianmeng** [comment]
@tleonhardt @kmvanbrunt Sorry guys, just saw your messages.

**Pull request closed without merging**